core/lib/config/src/configs/external_price_api_client.rs

```diff
@@ -0,0 +1,34 @@
+use std::time::Duration;
+
+use serde::Deserialize;
+use url::Url;
+
+pub const DEFAULT_TIMEOUT_MS: u64 = 10_000;
+pub const DEFAULT_COINGECKO_API_URL: &str = "https://pro-api.coingecko.com";
```

Comment:
I don't think we should have a default here. It's better to have it in the coingecko client impl.